nextflow_schema.json

```diff
@@ -120,14 +119,22 @@
                     "type": "string",
                     "description": "Not necessary if genome is set. Which keytype to use for pathway analysis, e.g. ENSEMBL, not necessary if --run_pathway_analysis = false."
                 },
+                "run_pathway_analysis": {
+                    "type": "boolean",
+                    "description": "Turn on this flag if you wish to run pathway analysis."
+                },
                 "custom_gmt": {
                     "type": "string",
                     "description": "Path to custom GMT file for gost query, this allows e.g. to query older versions of databases for pathway analysis, not necessary if --skip_pathway_analysis = true."
                 },
-                "run_pathway_analysis": {
+                "set_background": {
                     "type": "boolean",
-                    "default": false,
-                    "description": "Turn on this flag if you wish to run pathway analysis."
+                    "default": true,
```

Comment:
Just a question for my understanding: shouldn't all parameters in nf-core pipelines be set to false as default, so you  always have the option to turn them on? (I remember this as a comment in the funcscan review).
Should be then something like `no_background` and `false`.
Changing it now might have many implications in other files, but maybe keep it in mind in case it is intended to turn into a nf-core pipeline.

Reply:
Hmm, I don't know about this...I usually just name the parameters in a way that is a bit simpler; I prefer `set_background = true` over `no_background = false` just to have a positive param instead of a double-negative param. And as mentioned above, deseq will not become nf-core :)